esi/src/lib.rs

```diff
@@ -1,25 +1,28 @@
-#![doc = include_str!("../README.md")]
+#![doc = include_str!("../../README.md")]
```

Comment:
Now that README is symlinked to the package directory, I'm not sure this is necessary. However, this seems to be working fine.

Reply:
It didn't compile before I did that.